src/components/List_Products.js

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect, useState } from 'react';
+import { getItems, saveItem } from '../lib/api';
+
+function List_Products() {
+  const [Items, setItems] = useState(null);
+  const [ItemName, setItemName] = useState(null);
+
+  /* Get Items */
+  useEffect(() => {
+    getItemsData();
+  }, []);
+  const getItemsData = async () => {
+    const Itm = await getItems();
+    setItems(Itm.docs);
+  };
+
+  /* Save Item */
+  const saveItemName = async () => {
+    await saveItem(ItemName);
+    getItemsData();
+  };
+
+  return (
+    <div>
+      <button onClick={saveItemName}>Save</button>
+      <input
+        type="text"
+        onChange={(e) => setItemName(e.target.value)}
+        placeholder="Name"
```

Comment:
Improvements:

Add `required` attribute, because you can have products with the name `empty`.

Reply:
Added this functionality (with a conditional).